docs/DeveloperGuide.md

```diff
@@ -125,20 +125,27 @@ Given below is the Sequence Diagram for interactions within the `Logic` componen
 The `Model`,
 
 * stores a `UserPref` object that represents the user’s preferences.
-* stores the address book data.
-* exposes an unmodifiable `ObservableList<Person>` that can be 'observed' e.g. the UI can be bound to this list so that the UI automatically updates when the data in the list change.
+* Stores the `Person`, `SalesRecordEntry` and `Ingredient` sub-components.
 * does not depend on any of the other three components.
 
 
-Given below is the class diagram showing details of the person model:
-
 The `Person` sub-component,
 * stores the address book data.
 * exposes an unmodifiable `ObservableList<Person>` that can be 'observed' e.g. the UI can be bound to this list so that the UI automatically updates when the data in the list change.
+* an employee is modelled as a person in tCheck
```

Comment:
Maybe put this as a note because the sentence doesn't make sense. It would have been read as
"The `Person` sub-component an employee is ..."

Reply:
Sure, I will move it into an additional info box. Thanks!